src/components/dialogs/network-modifications/line/creation/line-creation-dialog.js

```diff
@@ -349,13 +353,41 @@ const LineCreationDialog = ({
         [editData, studyUuid, currentNodeUuid, snackError, onCreateLine]
     );
 
-    const onValidationError = (errors) => {
+    //scroll to the first array error (limits) when submitting the form
+    useEffect(() => {
+        if (isSubmitting && errors && Object.keys(errors).length > 0) {
+            if (
+                tabIndexesWithError.includes(
+                    LineCreationDialogTab.LIMITS_TAB
+                ) &&
+                tabIndex === LineCreationDialogTab.LIMITS_TAB
+            ) {
+                let firstArrayWithErrors = Object.keys(errors[LIMITS])[0];
+                let temporaryLimitsErrors =
+                    errors[LIMITS][firstArrayWithErrors]?.[TEMPORARY_LIMITS];
+                if (temporaryLimitsErrors?.ref) {
+                    //case where ErrorInput is rendered
+                    let errorId = temporaryLimitsErrors.ref.name;
+                    let errorInput = document.getElementById(errorId);
+                    errorInput.scrollIntoView({
+                        behavior: 'smooth',
+                        block: 'nearest',
+                    });
+                }
```

Comment:
I don't think it would work as expected if there are other fields than those two arrays, it would scroll to arrays even if another field is in error

we can see it already if you add empty fields to 2nd table without typing any "ID"
If you press "Validate", it will scroll to the table error whereas the first error is for the "ID"